wiki/Beatmap/Difficulty_spread/en.md

```diff
@@ -0,0 +1,14 @@
+---
+tags:
+  - difficulty set
+  - progression
+---
+
+# Difficulty spread
+
+A **difficulty spread** is a set of [difficulties](/wiki/Beatmap/Difficulty) sorted by [star rating](/wiki/Beatmap/Star_rating) in ascending order, such that:
+
+- There are no missing [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) — every difficulty between the easiest and the hardest ones from the spread's range is present.
```

Comment:
technically you can have a spread with "missing" difficulties, basically you can omit any difficulty that's considered optional based on the drain time. for example, a 4:15 set that has easy-normal-insane-expert is rankable because the lowest diff required is insane, so you can omit any difficulty below that. (see [this](https://osu.ppy.sh/beatmapsets/1439897#taiko/2962574) and [this](https://osu.ppy.sh/beatmapsets/1452881#taiko/2987138) as example)

not sure if you want to cover this technicality or skip it.

Reply:
this is a valuable point, but I don't know how to insert into the existing sentence -- want to make up a suggestion?

Reply:
thoughts?
```suggestion
- There are no missing [difficulty levels](/wiki/Beatmap/Difficulty#difficulty-levels) — every required difficulty between the easiest and the hardest ones from the spread's range is present.
   - Difficulties lower than the lowest difficulty level required, as determined by [spread rules](/wiki/Ranking_Criteria#rules.1), can be considered as optional and excluded from spread requirements.
```